grammar.js

```diff
@@ -777,7 +777,7 @@ module.exports = grammar({
     character: $ => /\?(\\\S({[0-9]*}|[0-9]*|-\S([MC]-\S)?)?|\S)/,
 
     interpolation: $ => seq(
-      '#{', $._statement, '}'
+     "#{", optional($._statement),"}"
```

Comment:
It looks like we use single quotes in this file, best keep it consistent.
```suggestion
     '#{', optional($._statement),'}'
```

Reply:
There are places where `"` are being used, but I changed given it's indeed more common.